package.json

```diff
@@ -4,43 +4,66 @@
   "description": "Proxy that handles interactions with pagoPA",
   "main": "index.ts",
   "repository": "https://github.com/teamdigitale/italia-pagopa-proxy",
-  "author": "Federico Feroldi <federico@teamdigitale.governo.it>",
+  "author": "Sergio Manetta <sergio.manetta@it.ibm.com>",
   "license": "MIT",
   "scripts": {
-    "build": "npm run lint && tsc",
-    "test": "jest",
+    "build": "tslint --project . && tsc",
+    "clean": "rm -r ./lib",
```

Comment:
if lib is the destination dir for compiled javascript files, please rename it to "dist"